workflows/prognostic_c48_run/tests/test_config.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from runtime.config import get_model_urls
+
+
+@pytest.mark.parametrize(
+    "config, model_urls",
+    [
+        ({}, []),
+        ({"scikit_learn": None, "prephysics": None}, []),
+        ({"scikit_learn": None, "prephysics": [{"some_prescribed_data": 0}]}, []),
+        (
+            {
+                "scikit_learn": {"model": ["ml_model_url"]},
+                "prephysics": [{"some_prescribed_data": 0}],
+            },
+            ["ml_model_url"],
+        ),
+        (
+            {
+                "scikit_learn": {"model": ["ml_model_url"]},
+                "prephysics": [
+                    {"some_prescribed_data": 0},
+                    {"model": ["prephysics_model_0", "prephysics_model_1"]},
+                ],
+            },
+            ["ml_model_url", "prephysics_model_0", "prephysics_model_1"],
+        ),
+    ],
+)
+def test_get_model_urls(config, model_urls):
```

Comment:
Since it's coupled to the `UserConfig`  for now, maybe use that class in the test so if one-or the other changes, this test will break.